src/wallet/spend.cpp

```diff
@@ -940,7 +940,9 @@ static util::Result<CreatedTransactionResult> CreateTransactionInternal(
 
     // The only time that fee_needed should be less than the amount available for fees is when
     // we are subtracting the fee from the outputs. If this occurs at any other time, it is a bug.
-    assert(coin_selection_params.m_subtract_fee_outputs || fee_needed <= nFeeRet);
+    if (!coin_selection_params.m_subtract_fee_outputs && fee_needed > nFeeRet) {
+        return util::Error{STR_INTERNAL_BUG("Fee needed > fee required")};
```

Comment:
```suggestion
        return util::Error{Untranslated(STR_INTERNAL_BUG("Fee needed > fee required"))};
```

might have to mark untranslated?

Reply:
Fixed